Time to play:
Loading lists of data
graph data Data(x=[81, 1], edge_index=[2, 324], y=[81], pos=[81, 2])
Traceback (most recent call last):
  File "/home/dami/Inverse_GNN/train_FEMgpu.py", line 216, in <module>
    validate_loss = validate()
TypeError: validate() missing 1 required positional argument: 'fenicsopt'